app/models/customer.py

```diff
@@ -0,0 +1,25 @@
+from flask import current_app
+from app import db
+from sqlalchemy.orm import relationship
+
+
+class Customer(db.Model):
+    __tablename__ = "customer"
+    id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    name = db.Column(db.String)
+    postal_code = db.Column(db.Integer)
```

Comment:
This solution is fine but it causes the code to fail 2 smoke tests (there was inconsistency around the postal_code in the tests).